packages/http-server-csharp/eng/scripts/emit-scenarios.js

```diff
@@ -0,0 +1,278 @@
+/* eslint-disable no-console */
+import { execa } from "execa";
+import { copy, pathExists } from "fs-extra";
+import { mkdir, readFile, rm, writeFile } from "fs/promises";
+import { globby } from "globby";
+import inquirer from "inquirer";
+import ora from "ora";
+import pLimit from "p-limit";
+import { basename, dirname, join, resolve } from "path";
```

Comment:
```suggestion
import { basename, dirname, join, resolve } from "pathe";
```

use this pacakge to not have to deal with windows `\`